### DISCUSSION TITLE
Clean up Wiki [done]

### DISCUSSION THREAD

**Discussion by bittner** [behave/behave#883] [Show and tell]
We can probably delete the [How to compile docs in HTML format](https://github.com/behave/behave/wiki/How-to-compile-docs-in-HTML-format%3F) in the Wiki.

This is all covered by our Tox configuration. The page is likely obsolete.

```
tox -e docs
```

**Comment by jenisys** [comment]
I actually think, I never used the Wiki ;-)

**Reply by bittner** [reply]
Okay, then let's delete _that_ page.

In addition, we should look into whether it makes sense to integrate the content of [Having issues with imports?](https://github.com/behave/behave/wiki/Having-issues-with-imports%3F) into our docs. Once this is done, we can delete also that page and disable the Wiki feature all together.

**Reply by bittner** [reply]
Deleted the page and moved the "Having ..." page to Home. :heavy_check_mark:

**Reply by bittner** [reply]
Disabled the Wiki. :checkered_flag:
